### PR TITLE
Feature jl sftp health check and Download Aris Update

### PR DESCRIPTION
**Pull request by JlevesqueAtBC**

### CODE REVIEW

File: backend/src/config/ssh2-sftp-client/index.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import Client from 'ssh2-sftp-client';
+
+const sftp = new Client();
+
+const config = {
+    host: process.env.LAN_FTP_SERVER_HOST || 'your.sftp.server',
+    port: process.env.LAN_FTP_SERVER_ || '22',
```
Comment (BradyMitch) [inline backend/src/config/ssh2-sftp-client/index.ts:7]:
```suggestion
    port: process.env.LAN_FTP_SERVER_PORT || '22',
```
?

File: backend/src/config/ssh2-sftp-client/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    username: process.env.LAN_FTP_SERVER__USER || "",
+    password: process.env.LAN_FTP_SERVER__PASSWORD || "",
```
Comment (BradyMitch) [inline backend/src/config/ssh2-sftp-client/index.ts:9]:
```suggestion
    password: process.env.LAN_FTP_SERVER_PASSWORD || "",
```
Same here

File: backend/src/config/ssh2-sftp-client/index.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+const remotePath = '/remote/path/';
```
Comment (BradyMitch) [inline backend/src/config/ssh2-sftp-client/index.ts:12]: We should just make this an environment variable

Comment (JlevesqueAtBC) [reply backend/src/config/ssh2-sftp-client/index.ts:12]: ok

**Pull request merged**